UniversityWebApplication/UniversityWebApplication/Views/ToDoCategories/Index.cshtml

```diff
@@ -15,21 +13,31 @@
             <th>
                 @Html.DisplayNameFor(model => model.CategoryName)
             </th>
-            <th></th>
+            <th>
+                Darbu sarasas
+            </th>
         </tr>
     </thead>
     <tbody>
-@foreach (var item in Model) {
-        <tr>
-            <td>
-                @Html.DisplayFor(modelItem => item.CategoryName)
-            </td>
-            <td>
-                <a asp-action="Edit" asp-route-id="@item.Category_Id">Edit</a> |
-                <a asp-action="Details" asp-route-id="@item.Category_Id">Details</a> |
-                <a asp-action="Delete" asp-route-id="@item.Category_Id">Delete</a>
-            </td>
-        </tr>
-}
+        @foreach (var item in Model)
+        {
+<tr>
+    <td>
+        @Html.DisplayFor(modelItem => item.CategoryName)
+    </td>
+    <td>
+        @*
+        @{ var list = "";}
+        @foreach (var listType in item.)
+        {
+
+        }*@

```

Comment:
Do we need this `<td>` now? If it has no special purpose - try not leave artefacts in code. It will get messy in the future